src/routers/books.js

```diff
@@ -1,4 +1,62 @@
-// Import data here...
+const express = require("express");
+const router = express.Router();
+const { books } = require("../../data");
+const {
+  checkForMissingFields,
+  checkForExistingEntry,
+  findById,
+} = require("../../src/utils/utils");
 
+// GET
 
-// Write routes here...
+router.get("/", (req, res) => {
+  // Gets all books
+  res.json({ books });
+});
+
+router.get("/:id", (req, res) => {
+  res.json({ book: findById(books, Number(req.params.id)) });
```

Comment:
I was going to say something about the client seeing a json response with `undefined` in it but then I noticed your `findById` function. Very nice implementation, it keeps this controller super small!